user/attachinfo/models.py

```diff
@@ -10,3 +10,5 @@ class AttachInfo(models.Model):
     location = models.CharField(max_length=MAX_LOCATION_LENGTH, blank=True)
     about = models.CharField(max_length=MAX_ABOUT_LENGTH, blank=True, default=DEFAULT_ABOUT)
     gravatar = models.CharField(max_length=MAX_GRAVATAR_LENGTH, blank=True)
+    codeforces = models.CharField(max_length=MAX_USERNAME_LENGTH, blank=True, default='errorerror')
```

Comment:
Default is 'errorerror'? 
IMO, leave blank may be a good choice.

Reply:
just a kidding........ http://codeforces.com/profile/errorerror